packages/server/src/auth/google.test.ts

```diff
@@ -204,31 +204,34 @@ describe('Google Auth', () => {
     expect(res2.body.code).toBeDefined();
   });
 
-  test('Custom Google client success', async () => {
```

Comment:
This test was an exact line-for-line duplicate of the test immediately above it.  Repurposing the real estate for new test.